Sxt1 mainpage vuejs

Pby-25 opened the pull request.

Pby-25 commented:
@1351530910 
Can you take a screenshot of what you mean by the books are not displaying?
As for the logout, I don't think it was ever implemented on this page. I'll add it later

Pby-25 commented:
btw I noticed that the simpleserver is not serving the front-end on 127.0.0.1 when I'm using Firefox. This is weird since I remember it working when I pushed it yesterday.
Now I have to use localhost:8080 for the front-end to be served or use another browser like Chrome.

1351530910 commented:
@Pby-25 
<img width="1440" alt="screen shot 2018-02-28 at 3 23 59 pm" src="https://user-images.githubusercontent.com/24920242/36811432-cc3a6c72-1c9b-11e8-9d83-9d6408eb19fb.png">

1351530910 commented:
when I click logout or profile, it just refreshes the page. the only working button is add new book.

Pby-25 commented:
I added logout and profile page link to main.html. 
I also removed some jQuery that might be causing the books not showing up issue .

Can you check if it is still not working? @1351530910

1351530910 commented:
cover image is now showing correctly;
the personal information page does not display the current user;
logout does not work.

Pby-25 commented:
its normal that the profile page does not display current user, that's for a different task.
As for logout I just redirected the user from the main page to the login page, where the cache should be cleared (this will probably be a different task too), so it won't work on any other pages for now.
this task is just for displaying the mainpage properly using vuejs.

1351530910 commented:
i think we should create another branch just for vue.js because i dont think it is a good idea to break the original version which works perfectly for a not finished implementation.
